crates/core/src/lib.rs

```diff
@@ -1,3 +1,7 @@
+// This feature flag allows us to exclude the unit tests from the coverage
+// report, so they do not skew the metrics of line coverage
+#![feature(coverage_attribute)]
+
```

Comment:
So this is a nightly feature right now, which means we either need to remove it until it becomes stable, ooooor add a toolchain file - https://rust-lang.github.io/rustup/overrides.html#the-toolchain-file

Reply:
Or add a nightly feature and use cfg_attr!